PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/layers/transmitter.cpp

```diff
@@ -0,0 +1,18 @@
+#include "transmitter.h"
+
+void Transmitter::sendInitFC1(void) {
+	this->layers->sendNOPFlit(this->globals, Dllp::DllpType::initFC1, this->sendOn);
+}
+
+void Transmitter::sendInitFC2(void) {
+	this->layers->sendNOPFlit(this->globals, Dllp::DllpType::initFC2, this->sendOn);
+}
+
+void Transmitter::sendSequence(std::vector<TLP*> sequence) {
+	std::queue<TLP*> tlpQueue;
+	
+	// copy the sequence into a queue of TLPs
+	for (auto it = sequence.begin(); it != sequence.end(); ++it) { tlpQueue.push(*it); }
```

Comment:
If all this does is change from a vector to a queue, I would suggest changing this function to accept a queue instead of a vector and modify the calling function